odoo/tests/common.py

```diff
@@ -1498,7 +1498,7 @@ def _wait_code_ok(self, code, timeout, error_checker=None):
 
     def navigate_to(self, url, wait_stop=False):
         self._logger.info('Navigating to: "%s"', url)
-        nav_result = self._websocket_request('Page.navigate', params={'url': url}, timeout=20.0)
+        nav_result = self._websocket_request('Page.navigate', params={'url': url}, timeout=40.0)
```

Comment:
20 seconds looks already to much to load a page especially now that assets are pregenerate. Do you have an example of a test close to this limit?